ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/DescribeConnectorExecutor.java

```diff
@@ -99,7 +104,12 @@ public StatementExecutorResponse execute(
       final ConnectResponse<Map<String, Map<String, List<String>>>> topicsResponse = serviceContext
           .getConnectClient()
           .topics(connectorName);
-      if (topicsResponse.error().isPresent()) {
+      if (topicsResponse.error().isPresent()
+          && topicsResponse.httpCode() == HttpStatus.SC_NOT_FOUND) {
```

Comment:
Can we add a quick comment in the code about why we handle the 404 case differently? I can imagine future readers seeing this and being confused.

Reply:
Done in 9eda284.